src/extension.ts

```diff
@@ -99,6 +101,8 @@ let helpAndFeedbackView: VSCode.TreeView<HelpAndFeedbackLink>;
 let taintVulnerabilityCollection: VSCode.DiagnosticCollection;
 const currentProgress: Record<string, { progress: VSCode.Progress<{ increment?: number }>, resolve: () => void } | undefined> = {};
 
+ContextManager.init();
```

Comment:
Could we imagine to make the `_instance` lazy-initialized in `ContextManager` instead?